src/js/libgeo/GeoOps.js

```diff
@@ -2353,6 +2359,9 @@ geoOps.TrAffine.updatePosition = function(el) {
 geoOps.TrSimilarity = {};
 geoOps.TrSimilarity.kind = "Tr";
 geoOps.TrSimilarity.signature = ["P", "P", "P", "P"];
+geoOps.TrSimilarity.initialize = function(el) {
+    el.isEuclidean = 1;
```

Comment:
The term `Euclidean` is probably inappropriate here. Essentially this is referring to a transformation which fixes the ideal circle points I and J. That's essentially the intersection between generalized Möbius transformations and projective transformations. The term comes from Cinderella terminology. Would `isSimilarity` be easier to understand?